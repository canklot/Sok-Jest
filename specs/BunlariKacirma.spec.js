import TestComponent from '../components/BunlariKacirma.vue'
import { mount } from '@vue/test-utils'

test('snapshot kontrolu', () => {
    const wrapper = mount(TestComponent)
    expect(wrapper).toMatchSnapshot()
  })

 /*  urun sayisini say */
 test('mount a vue component', () => {
  const wrapper = mount(TestComponent)
  const lis = wrapper.findAll('div.urun');  /* kactane oldugunu sayiyor */
  expect(lis.length).toBe(6);
})

 