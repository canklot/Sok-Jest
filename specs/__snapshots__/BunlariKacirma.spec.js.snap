// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount a vue component 1`] = `
<div class="BunlariKacirma">
  <div class="Baslik">Bunları Kaçırmayın</div>
  <div class="urun"><img src="@/images/lipton.jpg" class="urunResim">
    <div class="indirim">25 TL üzeri indirimli</div>
    <div class="urunAdi">Lipton Altın Harman Demşik Poşet 100'lü</div>
    <div class="eskiFiyat"><del>₺43,90</del></div>
    <div class="yeniFiyat">₺22,00</div>
    <div class="arti">+</div>
  </div>
  <div class="urun"><img src="@/images/deterjan.jpg" class="urunResim">
    <div class="indirim">25 TL üzeri indirimli</div>
    <div class="urunAdi">Bingo Sıvı Deterjan 40 Yıkama 2,6Lt</div>
    <div class="eskiFiyat"><del>₺43,90</del></div>
    <div class="yeniFiyat">₺22,00</div>
    <div class="arti">+</div>
  </div>
  <div class="urun"><img src="@/images/vanish.jpg" class="urunResim">
    <div class="indirim">25 TL üzeri indirimli</div>
    <div class="urunAdi">Vanish Oxi Action Gold Leke Çıkarıcı 1,4Lt</div>
    <div class="eskiFiyat"><del>₺32,90</del></div>
    <div class="yeniFiyat">₺18,00</div>
    <div class="arti">+</div>
  </div>
  <div class="urun"><img src="@/images/nivea.jpg" class="urunResim">
    <div class="indirim">25 TL üzeri indirimli</div>
    <div class="urunAdi">Nivea Soft Bakım Kremi Nemlendirici 200Ml</div>
    <div class="eskiFiyat"><del>₺16,95</del></div>
    <div class="yeniFiyat">₺10,00</div>
    <div class="arti">+</div>
  </div>
  <div class="urun"><img src="@/images/deterjan.jpg" class="urunResim">
    <div class="indirim">25 TL üzeri indirimli</div>
    <div class="urunAdi">Bingo Sıvı Deterjan 40 Yıkama 2,6Lt</div>
    <div class="eskiFiyat"><del>₺43,90</del></div>
    <div class="yeniFiyat">₺22,00</div>
    <div class="arti">+</div>
  </div>
  <div class="urun"><img src="@/images/deterjan.jpg" class="urunResim">
    <div class="indirim">25 TL üzeri indirimli</div>
    <div class="urunAdi">Bingo Sıvı Deterjan 40 Yıkama 2,6Lt</div>
    <div class="eskiFiyat"><del>₺43,90</del></div>
    <div class="yeniFiyat">₺22,00</div>
    <div class="arti">+</div>
  </div>
</div>
`;
